.devcontainer/Dockerfile

```diff
@@ -82,7 +81,11 @@ RUN GO_TARBALL="go${GOLANG_VERSION}.linux-amd64.tar.gz" \
  && curl https://artifactory.magmacore.org/artifactory/generic/${GO_TARBALL} --remote-name --location \
  && tar -xzf ${GO_TARBALL} \
  && rm ${GO_TARBALL}
-ENV PATH=${PATH}:/usr/local/go/bin:${HOME}/go/bin
+ENV PATH=${PATH}:/usr/local/go/bin
+
+# /home/vscode/go/bin doesn't exist initially, but for example orc8r/cloud/go/Makefile
+# populates that folder and expects those binaries to be in PATH
+ENV PATH=${PATH}:/home/vscode/go/bin
```

Comment:
I split this into an extra step to let the Go installation in lines 78 to 84 stand for itself.